packages/server/src/util/test-utils/data/observation-generator.ts

```diff
@@ -0,0 +1,22 @@
+import { Observation } from '../../../api/observation/observation.model';
```

Comment:
It looks like this file got added back in, but the function isn't used. I had pulled it to avoid confusion (since I fell victim to it for a moment). It should be safe to delete.